packages/terra-tag/tests/jest/RollUpTag.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { mountWithIntl } from 'terra-enzyme-intl';
+import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable import/no-extraneous-dependencies */
+import { RollupTag } from 'terra-tag';
```

Comment:
Does this use the local source code?

Reply:
Updated the import path points to the local. Thanks